src/shared/check_outdated_data.h

```diff
@@ -0,0 +1 @@
+int check_outdated_data(int maxTimeDiff, char *receivingData);
```

Comment:
This file should be called checkOutdatedData.h to comply with #3

You also need a define header pattern like this
```c
#ifndef CHECK_OUTDATED_DATA_H
#define CHECK_OUTDATED_DATA_H

int check_outdated_data(int maxTimeDiff, char *receivingData);

#endif
```

Reply:
yes sir